components/generic/HeroImage.vue

```diff
@@ -1,5 +1,48 @@
 <template>
+  <b-container
+    v-if="compact"
+    fluid
+    class="hero-wrapper"
+  >
+    <h1>{{ header }}</h1>
+    <p class="lead">
+      {{ lead }}
+    </p>
+    <SocialShare
+      :media-url="imageUrl"
+    />
+    <b-jumbotron
+      :style="jumbotronStyle"
+      fluid
+      text-variant="white"
+      data-qa="hero banner"
+      class="mt-4"
+      @click="citeCollapsed = true"
```

Comment:
Don't do this in the `@click` attributes themselves. Call event handler methods.

Reply:
ok, moved to method